drydock/templates/drydock/task/mongodb/init

```diff
@@ -7,15 +7,15 @@ mongosh --host {% if MONGODB_REPLICA_SET %}{{ MONGODB_REPLICA_SET }}/{% endif %}
         pwd:  "{{ MONGODB_PASSWORD }}",
         roles: [
           { role: "readWrite", db: "{{ MONGODB_DATABASE }}" },
-          {% if FORUM_DOCKER_IMAGE is defined %}{ role: "readWrite", db: "{{ FORUM_MONGODB_DATABASE }}" },{% endif %}
+          {% if 'forum' in PLUGINS  %}{ role: "readWrite", db: "{{ FORUM_MONGODB_DATABASE }}" },{% endif %}
         ]
     })
   } else {
     db.updateUser("{{ MONGODB_USERNAME }}", {
       pwd: "{{ MONGODB_PASSWORD }}",
       roles: [
         { role: "readWrite", db: "{{ MONGODB_DATABASE }}" },
-        {% if FORUM_DOCKER_IMAGE is defined %}{ role: "readWrite", db: "{{ FORUM_MONGODB_DATABASE }}" },{% endif %}
+        {% if 'forum' in PLUGINS %}{ role: "readWrite", db: "{{ FORUM_MONGODB_DATABASE }}" },{% endif %}
```

Comment:
Was the `PLUGINS` variable always available or was it added in the Tutor Sumac release?

Reply:
Always been available